openapi/src/test/groovy/io/micronaut/openapi/visitor/OpenApiIncludeVisitorSpec.groovy

```diff
@@ -29,24 +31,10 @@ import io.swagger.v3.oas.annotations.responses.ApiResponse;
                 license = @License(name = "Apache 2.0", url = "https://foo.bar"),
                 contact = @Contact(url = "https://gigantic-server.com", name = "Fred", email = "Fred@gigagantic-server.com")
         ),
-        tags = {
-                @Tag(name = "Tag 1", description = "desc 1", externalDocs = @ExternalDocumentation(description = "docs desc")),
-                @Tag(name = "Tag 2", description = "desc 2", externalDocs = @ExternalDocumentation(description = "docs desc 2")),
-                @Tag(name = "Tag 3")
-        },
```

Comment:
Removed because this test doesn't check anything related to `tags` (that is done in other tests)